### PR TITLE
feat: add mirror endpoint

### LINKED ISSUE #692: Create Add Mirror Endpoint

### PR DESCRIPTION
**Pull request by ec2sw**
fix #692

### CODE REVIEW

File: src/datalayer/writeService.js [hunk 1]
```diff
@@ -36,6 +36,23 @@ const createDataLayerStore = async () => {
   return storeId;
 };
 
+const updateDataLayerStore = async (storeId, { mirrorUrl }) => {
```
Comment (MichaelTaylor3D) [inline src/datalayer/writeService.js:39]: this should be called addMirror

File: src/datalayer/writeService.js [hunk 2]
```diff
@@ -36,6 +36,23 @@ const createDataLayerStore = async () => {
...
+  if (USE_SIMULATOR) {
+    logger.info('Updating data layer in simulator has no effect. Ignoring request.');
+    return;
+  }
+
+  logger.info(
+    `Update storeId: ${storeId}, waiting for this to be confirmed on the blockchain.`,
+  );
+  await waitForStoreToBeConfirmed(storeId);
```
Comment (MichaelTaylor3D) [inline src/datalayer/writeService.js:48]: this is unneeded

**Comment by ec2sw** [comment]
Added Postman collection for addMirror. Will add remove mirror postman next

**Comment by ec2sw** [comment]
manual merge conflicts

**Pull request merged**